### PR TITLE
DRAFT Compiler FE: Support DepthToSpace operation

### PR DESCRIPTION
**Pull request by d-krylov**
Support DepthToSpace operation

ONE-DCO-1.0-Signed-off-by: Denis Krylov <krylovdenism@gmail.com>

### CODE REVIEW

File: compiler/luci/export/src/CircleOperationExporter.cpp [hunk 1]
```diff
@@ -343,6 +344,20 @@ void OperationExporter::visit(luci::CircleCustom *node)
...
 
+void OperationExporter::visit(luci::CircleDepthToSpace *node)
+{
+  uint32_t op_idx = md.registerBuiltinOpcode(circle::BuiltinOperator_DEPTH_TO_SPACE);
+  std::vector<int32_t> inputs_vec{get_tensor_index(node->input())};
+  std::vector<int32_t> outputs_vec{get_tensor_index(static_cast<loco::Node *>(node))};
+
+  auto inputs = builder.CreateVector(inputs_vec);
+  auto outputs = builder.CreateVector(outputs_vec);
+  auto options = CreateDepthToSpaceOptions(builder);
```
Comment (seanshpark) [inline compiler/luci/export/src/CircleOperationExporter.cpp:355]: `CreateDepthToSpaceOptions` requires parameters...

File: compiler/luci/lang/src/Nodes/CircleDepthToSpace.test.cpp [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+TEST(CircleDepthToSpaceTest, constructor_P)
+{
+  luci::CircleDepthToSpace std_node;
+
+  ASSERT_EQ(luci::CircleDialect::get(), std_node.dialect());
+  ASSERT_EQ(luci::CircleOpcode::DEPTH_TO_SPACE, std_node.opcode());
+
+  ASSERT_EQ(nullptr, std_node.input());
+}
```
Comment (seanshpark) [inline compiler/luci/lang/src/Nodes/CircleDepthToSpace.test.cpp:31]: Can you add test for `block_size` attribute also ?

**Comment by seanshpark** [comment]
Seems OK with some change requests. Please proceed with @binarman 's approval

**Comment by seanshpark** [comment]
@binarman , can we close this?

**Comment by binarman** [comment]
@seanshpark 
Yes, everything is done. closing

**Pull request closed without merging**